src/main/resources/messages/messages.properties

```diff
@@ -485,7 +485,7 @@ message.warning.not_yml=File @|bold '%s'|@ is not a YAML or YML file
 message.warning.browser_not_found=\nError opening a web browser. Please open the following link manually:\n@|bold %s|@
 message.warning.file_not_uploaded_cause_of_language=Translation file @|yellow,bold '%s'|@ @|yellow hasn't been uploaded|@ since @|bold %s|@ is not enabled as a target language for the source file in your Crowdin project
 message.warning.auto_approve_option_with_mt='--auto-approve-option' is used only for the TM Pre-Translation method
-message.warning.no_file_to_download=Couldn't find any file to download
+message.warning.no_file_to_download=Couldn't find any file to download. As you are using the 'Skip untranslated files' option, please make sure you have fully translated files
```

Comment:
it should be a separate message

the `message.warning.no_file_to_download` should be used as before, and a new message should be used in the specific case when the skip untranslated files were used